cypress/e2e/0-api/configuration_api.cy.js

```diff
@@ -0,0 +1,377 @@
+describe('configuration API', () => {
+  const PORT_REGEX = /[0-9]*/;
+
+  beforeEach(() => {
+    cy.login();
+  });
+
+  context('GET /config/network', () => {
+    it('retrieve network settings', () => {
+      cy.reqNetwork().then(response => {
+        const body = response.body;
+        expect(body.type).to.be.a('string');
+        if (body.type !== 'DHCP') {
+          expect(body).to.have.property('address');
+          if (body.address !== null) { expect(body.address).to.be.a('string'); }
+          expect(body).to.have.property('subnetMask');
+          if (body.subnetMask !== null) { expect(body.subnetMask).to.be.a('string'); }
+          expect(body).to.have.property('gateway');
+          if (body.gateway !== null) { expect(body.gateway).to.be.a('string'); }
+          expect(body).to.have.property('dns');
+          if (body.dns !== null) { expect(body.dns).to.be.a('string'); }
+        }
+      });
+    });
+  });
+
+  context('GET /config/hostname', () => {
+    it('retrieve hostname', () => {
+      cy.reqHostname().then(response => {
+        const body = response.body;
+        expect(body.hostname).to.be.a('string');
+      });
+    });
+  });
+
+  context('GET /config/ntp', () => {
+    it('retrieves NTP client settings', () => {
+      cy.reqNTPClient().then(response => {
+        const body = response.body;
+        body.ntpServers.forEach(serverName => {
+          expect(serverName).to.be.a('string');
+        });
+        expect(body.timezone).to.be.a('string');
+      });
+    });
+
+    it('retrieves NTP client status', () => {
+      cy.reqNTPClientStatus().then(response => {
+        const body = response.body;
+        expect(body.log).to.be.a('string');
+      });
+    });
+  });
+
+  context('POST /config/ntp', () => {
+    const NTP_SERVERS = ['0.voltserver.pool.ntp.org', '1.voltserver.pool.ntp.org', '2.voltserver.pool.ntp.org', '3.voltserver.pool.ntp.org', '4.voltserver.pool.ntp.org'];
```

Comment:
I would suggest using non-default values here.  A good example would be 

```suggestion
    const NTP_SERVERS = [
      'time-a-g.nist.gov',
      'pool.ntp.org', 
      'time.google.com',
    ];
```